Class-2-(ControlFlow-&-Collections).md

```diff
@@ -2,40 +2,41 @@
-    1. Ask the class to give some examples of similarities or differences for conditionals and loops in csharp vs. other coding languages.
-1. The "fallthrough" aspect of ``switch`` statements is not always obvious and should be reviewed. However, do not sink too much time into this, since the students will use ``if/else`` statements much more often than ``switch``.
+1. Encouragement: Point out that learning a second coding language is usually easier than learning the first. The students already know how to use loops and conditionals, so applying them in C# is just a matter of recognizing the different syntax.
+    1. Ask the class to give some examples of similarities or differences for conditionals and loops in C# vs. other coding languages.
+1. The *fallthrough* aspect of ``switch`` statements is not always obvious and should be reviewed. However, do not sink too much time into this, since the students will use ``if/else`` statements much more often than ``switch``.
 1. Ask the students to explain the difference between ``while`` and ``do...while`` loops.
 1. Review how to use ``break`` and ``continue`` statements within loops.
-1. Review ``for...each`` loops in the context of Array and ArrayList collections. Note
-how the syntax changes when applied to HashMaps.
-1. As much as possible, live-code examples for ArrayLists and HashMaps, including how to use the most common methods.
+1. Review ``for...each`` loops in the context of array and list collections. Note
+how the syntax changes when applied to dictionaries.
+1. As much as possible, live-code examples for lists and dictionaries, including how to use the most common methods.
 
-### Class 2 Studio (TF Notes)
+### Class 2 Studio (TA Notes)
 
 1. With your group, discuss the "Some Items to Ponder Before Starting" section of the studio instructions.
 1. Remember to check in with EVERY student during the studio time.
     1. Ask them detailed questions about studio 1 if they did not finish it before the end of the last class.
     1. As they code studio 2, ask them to explain what different portions of their code does.
 1. Common mistakes:
-    1. Not properly setting up the ``main`` method.
-    1. Forgetting to import the necessary classes (e.g. HashMap).
-    1. Even though HashMaps are the better option, many students will try to use Arrays or ArrayLists because they see them as being easier. While the students could make these collections work, be prepared to nudge them in a different direction if they get stuck.
+    1. Not properly setting up the ``Main`` method.
+    1. Forgetting to import the necessary classes (e.g. List).
+    1. Even though Dictionaries are the better option, many students will try to use arrays or lists because they see them as being easier. While the students could make these collections work, be prepared to nudge them in a different direction if they get stuck.
```

Comment:
Maybe instead of "nudge them in a different direction if they get stuck", "be prepared to talk about the differences between the different types of collections and why dictionaries would be the better option."